server/migrations/1653662298756_validate-latest-ros-participant-entry.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable camelcase */
+const { dbClient, collections } = require('../db');
+
+exports.shorthands = 'validate-latest-ros-participant-entry';
```

Comment:
Just out of curiosity : What does exports.shorthands do?

Reply:
`exports.shorthands` is autogenerated code ...so, I added the migration name here this will provide some more readable logs.